app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -136,6 +136,10 @@ description: |
   elsif nic_delay
     options.push("nicdelay=#{nic_delay} linksleep=#{nic_delay}")
   end
+
+  if host_param('append_kernel_options')
+    options.push("#{host_param('append_kernel_options')}")
```

Comment:
you don't have to use the interpolation here, e.g. just `options.push(host_param('append_kernel_options'))` should work